app/templates/profile.hbs

```diff
@@ -1,3 +1,13 @@
-{{#travis-layout layoutName="layouts/profile" class="profile-view"}}
-  {{outlet}}
-{{/travis-layout}}
+{{#if showLoading}}
+  {{#travis-layout layoutName="layouts/center"}}
+    {{loading-screen center=true margin=true}}
+  {{/travis-layout}}
+{{else if showProfile}}
```

Comment:
I’m guessing this kind of pseudo-routing work is because the URLs for unsubscribing are nested here but you don’t want all the profile things to show here. I wonder if it’s worth noting that somewhere? Maybe both here and in the controller?